src/oc_erchef/rebar.config

```diff
@@ -15,29 +15,29 @@
     {chef_authn, ".*",
         {git, "git://github.com/chef/chef_authn.git",                 {branch, "master"}}},
     {chef_secrets, ".*",
-        {git, "git://github.com/chef/chef_secrets.git",               {branch, "master"}}},
+        {git, "git://github.com/chef/chef_secrets.git",               {branch, "lbaker/ej"}}},
     {darklaunch, ".*",
         {git, "git://github.com/chef/opscode-darklaunch-erlang.git",  {branch, "master"}}},
     {edown, ".*",
         {git, "git://github.com/uwiger/edown.git",                    {branch, "master"}}},
     {efast_xs, ".*",
         {git, "git://github.com/chef/efast_xs.git",                   {branch, "master"}}},
     {ej, ".*",
-        {git, "git://github.com/chef/ej.git",                         {branch, "lbaker/fixes-for-erlang19"}}},
+        {git, "git://github.com/chef/ej.git",                         {branch, "master"}}},
     {envy, ".*",
         {git, "git://github.com/markan/envy.git",                     {branch, "master"}}},
     {eper, ".*",
         {git, "git://github.com/massemanet/eper.git",                 {branch, "master"}}},
     {erlcloud, ".*",
         {git, "git://github.com/chef/erlcloud.git",                   {branch, "lbaker/presigned"}}},
     {erlware_commons, ".*",
-        {git, "https://github.com/chef/erlware_commons.git",          {branch, "lbaker/fix_for_ftmap"}}},
+        {git, "git://github.com/chef/erlware_commons.git",            {branch, "lbaker/fix_for_ftmap"}}},
     {folsom, ".*",
         {git, "git://github.com/boundary/folsom.git",                 {tag, "0.8.2"}}},
     {folsom_graphite, ".*",
         {git, "git://github.com/chef/folsom_graphite.git",            {branch, "master"}}},
     {ibrowse, ".*",
-        {git, "git://github.com/chef/ibrowse.git",                    {branch, "chef-server"}}},
+        {git, "git://github.com/chef/ibrowse.git",                    {branch, "master"}}},
```

Comment:
I do not think we want to change this: https://github.com/cmullaparthi/ibrowse/compare/master...chef:chef-server We want these changes or we see intermittent failures.

Reply:
We can keep pointing to the chef-server branch.

Reply:
Done.